apyros/metalog.py

```diff
@@ -57,7 +71,11 @@ def getLog( self, prefix ):
                 ret = line.split()[1].strip()
                 assert ret.startswith("logs/")
                 del self.lines[:i+1]  # cut already passed lines
-                return os.path.dirname( self.filename ) + os.sep + ret[4:]
+                if self.zf is None:
+                    return os.path.dirname( self.filename ) + os.sep + ret[5:]
```

Comment:
with change to zip file I would use all filenames relative to metalog as you are suggesting and remove directory paths completely ... so at the moment it is "for compatibility", but the future plan is clear.